builder/llpcBuilderImplInOut.cpp

```diff
@@ -348,10 +349,20 @@ void BuilderImplInOut::MarkGenericInputOutputUsage(
 
     if ((isOutput == false) || (m_shaderStage != ShaderStageGeometry))
     {
+        bool keepAllLocations = false;
```

Comment:
I can check if the internal tests that I have cover those cases.  Do you have a specific concern?  All this does is add all of the locations to the map instead of just the specific ones that have been used.  This could be better, but I don't see how it affects ClipDistance and CullDistance.

Reply:
I've checked and I do not see this as a problem.  I could be missing something, so we will keep it in mind.